mlflow/store/dbmodels/models.py

```diff
@@ -60,6 +60,8 @@ def _create_entity(base, model):
                     obj = SourceType.from_string(obj)
                 elif k == "status":
                     obj = RunStatus.from_string(obj)
+                elif k == "experiment_id":
+                    obj = str(obj)
```

Comment:
The main fix - also cast experiment ID to a string when deserializing it from the SQL DB.